parser.y

```diff
@@ -295,6 +295,7 @@ import (
 	block		"BLOCK"
 	booleanType	"BOOLEAN"
 	boolType	"BOOL"
+	bound		"BOUND"
```

Comment:
According to https://dev.mysql.com/doc/refman/8.0/en/keywords.html

BOUND is not a keyword, so it should be put to the NotKeywordToken section.

Reply:
Ok, I see. I thought none keywords go to UnReservedKeyword. Let me fix them.